src/__tests__/pages/policy/input/ParameterEditor.test.js

```diff
@@ -0,0 +1,52 @@
+// CenteredMiddleColumn.test.js
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import CenteredMiddleColumn from "/Users/introvertedbot/Documents/Projects/open-source-work/policyengine-app/src/layout/CenteredMiddleColumn.jsx";
```

Comment:
I think your import here isn't quite right. Could you import from within the repo?